examples/eth-pm/src/messaging/SendMessage.tsx

```diff
@@ -7,8 +7,7 @@ import {
   TextField,
 } from "@material-ui/core";
 import React, { ChangeEvent, useState, KeyboardEvent } from "react";
-import { Waku, WakuMessage } from "js-waku";
-import { hexToBytes } from "js-waku/lib/utils";
```

Comment:
You can use
```ts
import { hexToBytes } from "js-waku/build/esm/lib/utils";
```

Reply:
I don't think this is clearer as to where this is coming from. 

By importing `utils` from `js-waku` we're:
- being clear where it is coming from; and
- using the exported module in an idiomatic javascripty ESM way.

Reply:
```ts
import * as utils from "js-waku";

const { hextToBytes } = utils;
```